db/schema.rb

```diff
@@ -158,7 +158,7 @@
     t.jsonb "store"
     t.integer "reducible_id"
     t.string "reducible_type"
-    t.index ["id", "lock_version"], name: "index_subject_reductions_on_id_and_lock_version"
+    t.index ["id", "lock_version"], name: "index_subject_reductions_on_id_and_lock_version", unique: true
```

Comment:
I'm not sure these schema changes are valid. I assume these are local index testing changes / non-merged PR?

Reply:
that's.... weird? i ran the pending migrations and i assume this came from that? i'll pull it out.

Reply:
should be gone now.